src/dotnet-core-uninstall/Shared/Utils/Regexes.cs

```diff
@@ -53,29 +45,6 @@ internal static class Regexes
             $@"(?<{VersionGroupName}>{_notCapturedRuntimeVersionBasicRegexFormat})";
         private static readonly string _runtimeAuxVersionBasicRegexFormat =
             $@"(?<{AuxVersionGroupName}>{_notCapturedRuntimeVersionBasicRegexFormat})";
-        private static readonly Regex _sdkVersionDisplayNameRegex = new Regex(string.Format(
-            _sdkVersionBasicRegexFormat,
-            _previewVersionSdkDisplayNameRegex.ToString()));
-        private static readonly Regex _runtimeVersionDisplayNameRegex = new Regex(string.Format(
-            _runtimeVersionBasicRegexFormat,
-            _previewVersionRuntimeDisplayNameRegex.ToString()));
-        private static readonly Regex _aspNetRuntimeVersionDisplayNameRegex = new Regex(string.Format(
-            _runtimeVersionBasicRegexFormat,
-            _previewVersionAspNetRuntimeDisplayNameRegex.ToString()));
-        private static readonly Regex _hostingBundleVersionDisplayNameRegex = new Regex(string.Format(
-            _runtimeVersionBasicRegexFormat,
-            _previewVersionHostingBundleDisplayNameRegex.ToString()));
-        private static readonly Regex _hostingBundleAuxVersionDisplayNameRegex = new Regex(string.Format(
```

Comment:
these are removed due to applying corescan logic?

Reply:
Yes, they're no longer needed once we convert to corescan